Add new BBE for flush action (#3140)
ballerina-platform/ballerina-distribution: examples/flush/flush.md

> ## Flush

Author: ```suggestion
# Flush
```

> `flush` action is used by the workers to check wheather all the messages sent to a given worker are successfully recovered or not. If the transmission fails with an `error` or `panics`, then the error will be propagated to the waiting strand. Otherwise, it will return `nil`.

Reviewer: ```suggestion
The `flush` action is used by the workers to check whether all the messages sent to a given worker are successfully recovered or not. If the transmission fails with an `error` or `panics`, then, the error will be propagated to the waiting strand. Otherwise, it will return `nil`.
```

Reviewer 2: ```suggestion
The `flush` action is used by the workers to check whether all the messages sent to a given worker are successfully recovered or not. If the transmission fails with an `error` or a `panic`, then, the error will be propagated to the waiting strand. Otherwise, it will return `nil`.
```
